src/application/queries/query-models/conseillers.query-model.ts

```diff
@@ -30,3 +31,54 @@ export class DetailConseillerQueryModel {
   @ApiProperty()
   aDesBeneficiairesARecuperer: boolean
 }
+
+class BeneficiaireQueryModel {
+  @ApiProperty()
+  id: string
+
+  @ApiProperty()
+  nom: string
+
+  @ApiProperty()
+  prenom: string
+}
+
+export class ActionAQualifierQueryModel {
+  @ApiProperty()
+  @IsString()
+  id: string
+
+  @ApiProperty()
+  @IsString()
+  titre: string
+
+  @ApiProperty({ type: BeneficiaireQueryModel })
+  beneficiaire: BeneficiaireQueryModel
+
+  @ApiProperty()
+  @IsISO8601()
```

Comment:
t sur ? ya pas un is date string ??
bon après tout on check rien ici donc osef ça peut etre juste string

Reply:
Bah c'est ça, c'est date string au format iso